### PR TITLE
Set timeouts for shutting down the service

### PR DESCRIPTION
**Pull request by joehoyle**
This will allow running jobs to complete when trying to shutdown the service.

### CODE REVIEW

File: lib/Worker.php [hunk 1]
```diff
@@ -37,6 +38,8 @@ public function is_done() {
 	 * @return bool Did the process run successfully?
 	 */
 	public function shutdown() {
+
```
Comment (rmccue) [inline lib/Worker.php]: Excess newline

File: lib/Runner.php [hunk 2]
```diff
@@ -100,6 +100,7 @@ public function run() {
 
 	public function terminate( $signal ) {
```
Comment (joehoyle) [inline lib/Runner.php]: Well, you had a new line her?!

Comment (rmccue) [inline lib/Runner.php]: Oh, this shouldn't be here :)

**Comment by rmccue** [comment]
For context, from internal issue:
> by default Upstart and Systemd have a 5 second timeout when shutting down the daemon. This is often not long enough for a job to finish with the above fix, so I've changed that to 600 seconds. Most docs say that the hard power shutdown on EC2 is a total of about 10 minutes, so this should be ok.

**Pull request merged**